server/seeder.js

```diff
@@ -0,0 +1,31 @@
+const connection = require('./database.js');
+const mockImages = require('./s3mock.js');
+
+const getRandomImageURL = function(){
+  return mockImages[Math.floor((Math.random() * mockImages.length))];
+}
+
+const getRandomImageCount = function(){
+  return Math.floor(Math.random()*(5-3+1)+3);
+}
+
+const seedDatabase = function(){
+  //for 100 entries.
+  for (let i = 0; i < 100; i++){
+    let imageCount = getRandomImageCount();
+    let imageURLs = [];
+    for (let j = 0; j < imageCount; j++){
+      imageURLs.push(getRandomImageURL());
+    }
+    let query = `insert into images (path) VALUES ('${JSON.stringify(imageURLs)}');`

```

Comment:
Minor issue: This will always create items starting with id 1 and incrementing from there. It would be nice if the user could specify a range of ids--say, 100-200. (I seem to recall this being a requirement, but I might be wrong.)

Reply:
I think you're right. I'll make the adjustment accordingly.